blazor/data/how-to/change-update-type-from-PATCH-to-PUT.md

```diff
@@ -0,0 +1,56 @@
+---
+layout: post
+title: change the update type from PATCH to PUT in Blazor DataManager Component | Syncfusion
+description: Checkout and learn here all about changing the update type from PATCH to PUT in Syncfusion Blazor DataManager component and more.
+platform: Blazor
+control: DataManager
+documentation: ug
+---
+
+<!-- markdownlint-disable MD024 -->
+
+# Change the update type from PATCH to PUT
+
+In ODataV4, the PATCH method is the preferred means of updating data, and it is used for partial updates of data. So, to update the entire data collection we can use PUT method.
```

Comment:
OData v4 recommends using PATCH to update entities because it only changes the provided fields -  definition of PATCH

Reply:
frame new sentence using above